ProductListing/ProductListing.css

```diff
@@ -0,0 +1,53 @@
+.product-grid {
+  display: grid;
+  grid-template-columns: 20% 80%;
+}
+
+/* FILTERS BAR */
+
+.filters {
+  padding: 1rem;
+}
+
+.filter-heading {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.clear {
+  text-decoration: underline;
+}
+
+.filter-head {
+  margin: 1rem 0rem;
+}
+
+.filter-category {
```

Comment:
Maybe elements from filter category could have cursor pointer on hover? also maybe some sort of background color change one hover? could help UX I think

Reply:
Noted, will make the required changes.